shard.yml

```diff
@@ -1,5 +1,5 @@
 name: kemal
-version: 0.25.1
+version: 0.27.1
```

Comment:
This is actually Kemal version and not Crystal. 
So you should add another line as  `crystal: 0.27.1`

Reply:
Argghh, making commits at 🌃 ...